src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -25,212 +26,110 @@ class TunableService(CRUDService):
     class Config:
         datastore = 'system.tunable'
         datastore_prefix = 'tun_'
-        datastore_extend = 'tunable.upper'
         cli_namespace = 'system.tunable'
 
-    def __init__(self, *args, **kwargs):
-        super(TunableService, self).__init__(*args, **kwargs)
-        self.__default_sysctl = {}
+    SYSTEM_TUNABLES = deque()
 
     ENTRY = Patch(
         'tunable_create', 'tunable_entry',
         ('add', Int('id')),
     )
 
     @private
-    async def default_sysctl_config(self):
-        return self.__default_sysctl
+    def get_system_tunables(self):
+        if not TunableService.SYSTEM_TUNABLES:
```

Comment:
You could also use functools.cache and not maintain a separate variable for future

Reply:
I routinely forget about that decorator 😄